3 - PB/Documentazione esterna/Verbali/24-03-26/24-03-26.typ

```diff
@@ -0,0 +1,43 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "26/03/24",
+  subTitle: "Dimostrazione e convalida MVP",
+  docType: "verbale",
+  authors: (
+    "Nardo Silvio",
+  ),
+
+  externalParticipants : (
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+  ),
+
+  timeStart: "15:00",
+  timeEnd: "15:30",
+  location: "Zoom",
+);
+
+= Ordine del giorno
+- Presentazione MVP;
+- Pianificazione incontri futuri.
+
+== Presentazione MVP
+Sono state presentate al Proponente le diverse funzionalità implementate nell'MVP atte a rispettare i requisiti espressi nel documento #st_v, disponibile al link: #align(left,link("https://github.com/Error-418-SWE/Documenti/blob/main/3%20-%20PB/Documentazione%20esterna/Specifica%20Tecnica_v" + st_vo + ".pdf")) nella sezione "Requisiti soddisfatti".
```

Comment:
nel MVP*

Reply:
In teoria è corretto in entrambe le forme